site/app/controllers/admin/AdminGradeableController.php

```diff
@@ -155,7 +155,7 @@ private function editPage(Gradeable $gradeable, $semester, $course, $nav_tab = 0
             $repo_id_number++;
         }
         usort($all_repository_config_paths, function ($a, $b) {
-            return $a[0] > $b[0];
+            return strcmp($a, $b);
```

Comment:
Haven't looked too much at this but wouldn't we want this instead?
```suggestion
            return strcmp($a[0], $b[0]);
```